doc/en/api/1.0.0/gwclayers.yaml

```diff
@@ -0,0 +1,236 @@
+          schema:
+            $ref: "#/definitions/Layers"
+          examples:
+            application/xml: |
+              <layers>
+                <layer>
+                  <name>opengeo:countries</name>
+                  <atom:link xmlns:atom="http://www.w3.org/2005/Atom" rel="alternate" href="http://localhost:8080/geoserver/gwc/layers/opengeo%3Acountries.xml" type="text/xml"/>
+                </layer>
+                <layer>
+                  <name>usa:states</name>
+                  <atom:link xmlns:atom="http://www.w3.org/2005/Atom" rel="alternate" href="http://localhost:8080/geoserver/gwc/layers/usa%3Astates.xml" type="text/xml"/>
+                </layer>
+              </layers>
+
+  /layers/{layerName}:
+    get:
+      operationId: layersNameGet
+      summary: Retrieve a cached layer
+      description: Retrieves a single cached layer definition. Use the "Accept:" header to specify format or append an extension to the endpoint (example "/layers/{layer}.xml" for XML).
+      produces:
+        - application/xml
+      parameters:
+        - name: layerName
+          in: path
+          required: true
+          description: The name of the layer to retrieve.
+          type: string
+        - name: quietOnNotFound
+          in: query
+          required: false
+          description: The quietOnNotFound parameter avoids logging an exception when the layer  is not present. Note that 404 status code will still be returned.
+          type: boolean
+      responses:
+        200:
+          description: OK
+          schema:
+            $ref: "#/definitions/Layer"
+          examples:
+            application/xml: |
+              <GeoServerLayer>
+                <id>LayerInfoImpl--3a33e7d0:1400d3d823c:-7fdf</id>
+                <enabled>true</enabled>
+                <inMemoryCached>true</inMemoryCached>
+                <name>opengeo:countries</name>
+                <mimeFormats>
+                  <string>image/png</string>
+                  <string>image/jpeg</string>
+                </mimeFormats>
+                <gridSubsets>
+                  <gridSubset>
+                    <gridSetName>EPSG:900913</gridSetName>
+                  </gridSubset>
+                  <gridSubset>
+                    <gridSetName>EPSG:4326</gridSetName>
+                  </gridSubset>
+                </gridSubsets>
+                <metaWidthHeight>
+                  <int>4</int>
+                  <int>4</int>
+                </metaWidthHeight>
+                <expireCache>0</expireCache>
+                <expireClients>0</expireClients>
+                <parameterFilters>
+                  <styleParameterFilter>
+                    <key>STYLES</key>
+                    <defaultValue/>
+                  </styleParameterFilter>
+                </parameterFilters>
+                <gutter>0</gutter>
+              </GeoServerLayer>
+        404:
+          description: Rest Exception 404 Unknown layer
```

Comment:
Given that you alreay have 404 in the parent tag, just "Unknown Layer" is probably fine.